shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -49,6 +45,7 @@
  *  - {@link Activity#onRequestPermissionsResult(int, String[], int[])} ()}
  *  - {@link Activity#onNewIntent(Intent)} ()}
  *  - {@link Activity#onUserLeaveHint()}
+ *  - {@link Activity#onTrimMemory(int)}
```

Comment:
Here and below: similar comment to above: does this syntax work or should this be `<li>`?